frontend/src/pages/Status/StatusTable.js

```diff
@@ -70,6 +81,31 @@ const StatusTable = (props) => {
     isFetchingExportVersion
   } = props;
 
+  const [latestVersion, setLatestVersion] = useState(null);
+
+  useEffect(() => {
+    axios
```

Comment:
also, question here would be, if this request should be even called for non root users? If yes, should non root users alway get error in response?